spacy/gold.pyx

```diff
@@ -54,23 +51,25 @@ def tags_to_entities(tags):
     return entities
 
 
-def merge_sents(sents):
-    m_deps = [[], [], [], [], [], []]
-    m_cats = {}
+def merge_sents(raw_annots):
```

Comment:
Would it be metter to make this a classmethod constructor of `RawAnnot`?

Reply:
I think it should to be a method in the `DocAnnot`, which has an internal list of `RawAnnot`s, which this function should merge together into one long interinal `RawAnnot`.